FRC1076-2016-Competition/src/org/usfirst/frc/team1076/robot/physical/Robot.java

```diff
@@ -127,6 +127,14 @@ public void robotInit() {
 		armFollower.enableBrakeMode(true);
 		armExtendMotor.enableBrakeMode(true);
 		armExtendFollower.enableBrakeMode(true);
+
+		
+		armMotor.ConfigFwdLimitSwitchNormallyOpen(true);
+		armMotor.ConfigRevLimitSwitchNormallyOpen(true);
+		armMotor.enableLimitSwitch(true, true);
+		armExtendMotor.ConfigFwdLimitSwitchNormallyOpen(true);
+		armExtendMotor.ConfigRevLimitSwitchNormallyOpen(true);
+		armExtendMotor.enableLimitSwitch(true, true);
```

Comment:
We tried to make them follow in hardware, but we gave up at one point because there was some hardware problem. It's *should* only be a few dozen lines of code to add hardware following back in, but I don't know if it'll work as planned. To do it you have to set the control mode to follower, and `.set()` the motor to the id that it should follow.